monkey/monkey_island/cc/services/attack/technique_reports/__init__.py

```diff
@@ -47,51 +47,64 @@ def get_report_data():
         pass
 
     @staticmethod
-    def technique_status(technique):
+    def technique_status(tech_id):
```

Comment:
make `@classmethod`
get actual technique and not tech_id
inside function get tech_id from technique